src/compiler/gcc.rs

```diff
@@ -193,16 +196,17 @@ where
     // and interpreting it as a list of more arguments.
     let it = ExpandIncludeFile::new(cwd, arguments);
 
-    for item in ArgsIter::new(it, arg_info) {
+    for arg in ArgsIter::new(it, arg_info) {
+        let arg = try_arg!(arg.map_err(|e| e.static_description()));
```

Comment:
You seem to have this same `map_err` clause in all the places you use `try_arg!`, so why not just bake it into the macro?